src/e2e/java/teammates/e2e/cases/InstructorCourseDetailsPageE2ETest.java

```diff
@@ -116,7 +116,7 @@ public void testAll() {
         ______TS("delete student");
         detailsPage.sortByName();
         detailsPage.sortByStatus();
-        StudentAttributes[] studentsAfterDelete = { students[3], students[0], students[1] };
+        StudentAttributes[] studentsAfterDelete = { students[0], students[3], students[1] };
```

Comment:
could I check why the E2E test has changed?

Reply:
I believe this is due to the fact that in the original table, when clicking on sort, it would sort descending by default, while the current implementation would sort it in ascending by default, hence the change in the ordering